exporters/postgresql/postgresql_exporter.go

```
@@ -66,9 +64,8 @@ func (exp *postgresqlExporter) Init(cfg plugins.PluginConfig, logger *logrus.Log
 	}
 	exp.db = db
 	<-ready
-	if rnd, err := exp.db.GetNextRoundToAccount(); err == nil {
-		exp.round = rnd
-	}
+	importer.EnsureInitialImport(exp.db, *initProvider.GetGenesis())
+	exp.round = uint64(initProvider.NextDBRound())
```

You can return an error here if `NextDBRound() != GetNextRoundToAccount()`